### DISCUSSION TITLE
启动项目时报错，报错

### DISCUSSION THREAD

**Discussion by LuweeJia** [vbenjs/vue-vben-admin#5004] [Q&A]
![1733186907461](https://github.com/user-attachments/assets/7a22f0a6-014e-4920-993e-911c105dbc9e)

**Comment by likui628** [comment]
现在应该是没有了
https://github.com/vbenjs/vue-vben-admin/commit/24b9aa44d2c4aedd44323ccad6168b1552a2a05d
